resources/views/sidebar.blade.php

```diff
@@ -0,0 +1,21 @@
+@section('sidebar')
```

Comment:
I think it's good to have main folder for template 
like this
views/templates/sidebar.blade.php or views/parts/sidebar_template.blade.php

Reply:
Understood, sir.